app/grants/templates/grants/detail/template-grant-details.html

```diff
@@ -760,6 +786,170 @@ <h4 class="font-bigger-1 text-black">Help contributors discover your grant</h4>
           </b-form-checkbox>
         </div>
 
+        <!-- Chain -->
+        <div class="mb-4 col-12">
+          <hr>
+          <h5 class="mt-5 mb-4 font-weight-normal">Funding Details</h5>
+
+          <label class="font-subheader font-weight-light letter-spacing" for="bounty_chain">Select a blockchain</label>
+          
+          <v-select id="grant-details-edit-funding-details" class="grant-edit-disable-select" @input="resetAddress()" placeholder="Select a blockchain to receive funding" :options="grantChains" label="label" v-model="chainId" :reduce="state => state.name" disabled>
```

Comment:
why do we need a select field? not super clear here

Reply:
@thelostone-mc I have just used the same field as for new grants out of convenience. This field will never be enabled, I might as well remove the event handlers - I missed that.